nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -76,6 +76,8 @@
 
 - `zammad` has had its support for MySQL removed, since it was never working correctly and is now deprecated upstream. Check the [migration guide](https://docs.zammad.org/en/latest/appendix/migrate-to-postgresql.html) for how to convert your database to PostgreSQL.
 
+- `nodePackages."@commitlint/config-conventional"` has been removed, as it is a library, and your JS project should depend on it instead.
```

Comment:
```suggestion
- `nodePackages."@commitlint/config-conventional"` has been removed, as it is a library and projects should vendor it instead.
```

Reply:
Well, we're not talking about vendoring, but about depending on it(with npm, pnpm, etc) in this case, so I think "depend on" should be used, unless there's a term that would make more sense than either.